dash/frontend/src/app/core/dialogues/show-json-data/show-json-data.component.ts

```diff
@@ -54,8 +55,22 @@ export class ShowJsonDataComponent implements OnInit {
     this.falcoService.getFalcoLogs(this.clusterId,  { limit: this.limit, page: this.page, signature: this.signature})
       .pipe(take(1))
       .subscribe(response => {
-        this.dataSource = new MatTableDataSource(response.data.list);
+        const dataList = response.data.list;
         this.logCount = response.data.logCount;
+
+        // create a new data list without the current event log
+        for (let count = 0; count < this.logCount; count++){
```

Comment:
If you load the second page, does it need to "clear out" the previous page's data? 

Might be able to simplify this to... 

    this.newDataList = dataList.filter(i.id !== this.data.content.id)

Reply:
this.logAcount = response.data.logCount - 1;
    this.newDataList = dataList.filter(i.id !== this.data.content.id)